joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/cpp/features17/Cpp17FeaturesTests.scala

```diff
@@ -413,7 +469,7 @@ class Cpp17FeaturesTests extends AstC2CpgSuite(fileSuffix = FileDefaults.CppExt)
           |byte d = byte{1};
           |""".stripMargin)
       cpg.local.nameExact("b").typeFullName.l shouldBe List("byte")
-      cpg.identifier.nameExact("b").typeFullName.l shouldBe List("byte")
+      cpg.identifier.nameExact("b").typeFullName.l shouldBe List("byte", "byte")
```

Comment:
This is unexpected to me. What are the two identifiers doing here? Could you assert something in the test here that explains / shows what each of them is doing?

Reply:
One from the variable decl, one as base argument to the constructor call.